templates/errors/submission-complete.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'layouts/_base.html' %}
```

Comment:
This page has an exit button that doesn't link anywhere. The button only appear when the cookie session is still valid therefore we should link to the normal exit page which atm is the census home page.